posts/recap.mdx

```diff
@@ -17,7 +17,7 @@ excerpt: RECAP is an online archive and free extension for Firefox and Chrome th
 
 ## If you use PACER, install RECAP.
 
-Once installed, every docket or PDF you purchase on PACER will be added to the [RECAP Archive][archive]. Anything somebody else has added to the archive will be available to you for free — right in PACER itself.
+Once installed, every docket or PDF you purchase on PACER* will be added to the [RECAP Archive][archive]. Anything somebody else has added to the archive will be available to you for free — right in PACER itself.
```

Comment:
```suggestion
Once installed, every docket or PDF you purchase on district court PACER websites will be added to the [RECAP Archive][archive]. Anything somebody else has added to the archive will be available to you for free — right in PACER itself.
```

Reply:
Arg, yes! Thank you.

Reply:
It seemed more straightforward to list any exclusions *rather than*  to list all inclusions here, from a human-factors standpoint.